Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:10-alpine
+RUN apk add yarn
+RUN apk add openssh-client
+RUN apk add rsync
```

Comment:
```suggestion
FROM node:10-alpine

RUN apk add openssh-client rsync yarn
RUN mkdir /.yarn && touch /.yarnrc && chmod 777 /.yarn /.yarnrc
```

Reply:
If this does not work, we can instead bind-mount these file and folder from the host machine, but it requires more logic as they could not exist on the host too, and are in different places on Linux and Mac.